src/main/java/dev/watchwolf/entities/entities/EntityType.java

```diff
@@ -6,7 +6,7 @@
 import dev.watchwolf.entities.items.Item;
 
 public enum EntityType {
-    DroppedItem;
+    DROPPED_ITEM;
```

Comment:
Check on [line 18](https://github.com/rogermiranda1000/WatchWolf-Tester/blob/629f3155c00acad5c3d41c8cd80e5b14d688da8a/src/main/java/dev/watchwolf/entities/entities/EntityType.java#L18) I'll get the enum using the class name. You'll have to change the `getType` function to convert the enum name to UpperCamelCase.